src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -61,21 +122,36 @@ export default function Page() {
         setCurrentTimesheetsToDisplay (userTimesheets, date); 
     }
     
-    const [userTimesheets, setUserTimesheets] = useState([]); 
-    const [currentTimesheets, setCurrentTimesheets] = useState([]);
-    const [selectedTimesheet, setTimesheet] = useState();
+    //TODO: default to first employee but idk if employee always exists
+    const [selected, setSelected] = useState<UserSchema>();
+    const [user, setUser] = useState<UserSchema>();
+
+    const [userTimesheets, setUserTimesheets] = useState<TimeSheetSchema[]>([]); 
+    const [currentTimesheets, setCurrentTimesheets] = useState<TimeSheetSchema[]>([]);
+    const [selectedTimesheet, setTimesheet] = useState<TimeSheetSchema>();
     const columns = defaultColumns 
 
-    //Pulls user timesheets, marking first returned as the active one
+    // Pulls user timesheets, marking first returned as the active one
     useEffect(() => {
         // Uncomment this if you want the default one loaded 
         //setTimesheet(testingTimesheetResp)
+        apiClient.getUser().then(userInfo => {
+            setUser(userInfo);
+        })
         apiClient.getUserTimesheets().then(timesheets => {
```

Comment:
Let's make sure that there's proper docs about what we are expecting the `getUserTimesheets()` to return in apiClient - seems key to distinguish whose timesheets it's expected to return. 

Currently, I believe that endpoint just returns one user's timesheets. I know you have a TODO in apiClient to add a new endpoint in for supervisors/admins to get a requested user's timesheets, or edit the existing endpoint - maybe write a `TODO` here as well if a different endpoint will eventually need to be called.

Reply:
I redid how its being used. I changed it to take in a uuid and return the timesheets for that uuid. So it should return the timesheets for the given associate uuid. It should work the same for all roles however we need to add some guards so that associates can only request their own timesheets.